src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
I think you went a bit too far with that :) Differences aside, the important matter is that the concatenations might be avoided by using the logger context, and passing the needed key/values (although we have to trust that the logger implementation will be reasonable enough as to avoid any further processing when the log level is not appropiate). It's a bummer that PSR3 wont even return the current loglevel.

I'm not sure about